components/ord-service/src/test/resources/application.yml

```diff
@@ -56,6 +56,7 @@ destination_fetcher:
   reload_url: https://destination-fetcher/reload
   sensitive_data_url: https://destination-fetcher/sensitive-data
   tenant_header: Tenant
+  x_request_id_header: x-request-id
```

Comment:
This configuration is more general purpose and can be extracted outside of the destination_fetcher config.

This may be a prerequisite as well to remove the hardocded header name from filters.